src/Apps/Auction/Components/RegistrationForm.tsx

```diff
@@ -193,13 +109,15 @@ Yup.addMethod(Yup.string, "present", function(message) {
 })
 
 const validationSchema = Yup.object().shape({
-  name: Yup.string().present("Name is required"),
-  street: Yup.string().present("Address is required"),
-  country: Yup.string().present("Country is required"),
-  city: Yup.string().present("City is required"),
-  state: Yup.string().present("State is required"),
-  postalCode: Yup.string().present("Postal code is required"),
-  telephone: Yup.string().present("Telephone is required"),
+  address: Yup.object({
+    name: Yup.string().present("Name is required"),
+    addressLine1: Yup.string().present("Address is required"),
+    country: Yup.string().present("Country is required"),
+    city: Yup.string().present("City is required"),
+    region: Yup.string().present("State is required"),
+    postalCode: Yup.string().present("Postal code is required"),
+    phoneNumber: Yup.string().present("Telephone is required"),
```

Comment:
We're restructuring the validation rules here to account for the new values coming from the `AddressForm` component.